src/main/webapp/creatives/maps/simid_map.css

```diff
@@ -40,10 +70,11 @@ html, body {
  */
 #map {
     height: 80%;
-    width: 80%;
+    width: 100%;
```

Comment:
Is the change here from 80% to 100% just to make the map be "full screen" lengthwise, with a gap at the top & bottom? These dimensions might be worth going over together once the functionality is completed so we aren't overwriting each other's changes

Reply:
This is something we can figure out together once all of the pieces are done